src/SystemSettings/ListItems/ThinDivider.qml

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of system-settings
+ *
+ * Copyright (C) 2020 UBports Foundation <developers@ubports.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3, as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranties of
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Ubuntu.Components.ListItems 1.3 as ListItem
+
+ListItem.ThinDivider {
```

Comment:
Wouldn't you define this as:

```
import QtQuick 2.9
import Ubuntu.Components 1.3

Rectangle {
    width: parent.width
    height: units.dp(1)
    color: theme.palette.normal.base
}
```

or are you planning to change this ListItems dependency later on?

Reply:
Yes, I plan to get rid of this later on; for the time being I'm just trying to reduce the usage of obsolete items to the `src/SystemSettings/ListItems` module.

What would you say about "undeprecating" the ThinDivider in the UITK (and maybe implementing it as you are suggesting here)?

Reply:
Sounds like a plan :)

Oh, you mean «undeprecating» the ThinDivider but not the ListItems? Sounds good as we use it all over the system

Reply:
Yes! :-)